codalab/worker/worker.py

```diff
@@ -38,55 +38,52 @@ class Worker:
     NETCAT_BUFFER_SIZE = 4096
     # Number of seconds to wait for bundle kills to propagate before forcing kill
     KILL_TIMEOUT = 100
-    # Directory name to store running bundles in worker filesystem
```

Comment:
I omitted this constant now because it was added here after this PR was opened. We can re-add it as a constant to the main file, but it's a lot less jarring there as a hard-coded value than here.